treeherder/settings/base.py

```diff
@@ -206,9 +206,9 @@
 CELERY_DEFAULT_ROUTING_KEY = 'default'
 
 CELERYBEAT_SCHEDULE = {
-    'fetch-push-logs-every-minute': {
+    'fetch-push-logs-every-10-seconds': {
         'task': 'fetch-push-logs',
-        'schedule': timedelta(minutes=1),
+        'schedule': timedelta(seconds=10),
```

Comment:
This is fixing bug 1096919, rather than bug 1076826 - could we save that for bug 1096919?
I think it might be safer to change less at one time, and reducing the interval has additional dependencies (like the task locking):
https://bugzilla.mozilla.org/showdependencytree.cgi?id=1096919&maxdepth=1&hide_resolved=1